package.json

```diff
@@ -37,7 +37,7 @@
     "package:build:js": "cd $INIT_CWD && babel src --out-dir dist --source-maps --config-file \"../../babel.config.json\" --copy-files --no-copy-ignored --ignore \"**/__tests__\",\"**/__mocks__\"",
     "package:build:react-scripts": "cd $INIT_CWD && SKIP_PREFLIGHT_CHECK=true GENERATE_SOURCEMAP=false react-scripts build",
     "package:build:types": "cd $INIT_CWD && tsc",
-    "package:build:vite": "NODE_OPTIONS=--max-old-space-size=8192 cd $INIT_CWD && vite build --config \"../../vite.config.js\"",
```

Comment:
Backed out this config, since we were only really using it to fix out of memory issues